Uyanda.Coffee.Persistence/Accessors/BeverageAccessor.cs

```diff
@@ -67,6 +67,56 @@ public async Task<InvoiceModel> PurchaseAsync(IEnumerable<LineItemModel> lineIte
             return ToModel(invoice);
         }
 
+        public async Task<BeverageSizeCostModel[]> AddBeverageSizeCostAsync(BeverageSizeCostModel price)
+        {
+          
+            var beverageDictionary = await localDbContext.Beverages.AsNoTracking()
+                .Select(c => new { c.Id, c.Name }).ToDictionaryAsync(c=>c.Id, c=>c.Name);
+
+            if (!beverageDictionary.ContainsValue(price.Beverage.Name))
```

Comment:
Case and usage of name before Id might throw you off here.

Reply:
can you kindly clarify how